src/internet_identity/src/storage/anchor/tests.rs

```diff
@@ -138,6 +138,24 @@ fn should_enforce_cumulative_device_limit() {
     assert_eq!(anchor.devices().len(), 4);
 }
 
+#[test]
+fn should_enforce_cumulative_size_limit_on_identity_metadata() {
+    let mut anchor = Anchor::new();
+
+    let metadata = HashMap::from_iter(vec![(
+        "some key".to_string(),
+        MetadataEntry::String("a".repeat(3000)),
```

Comment:
If you decide not to have separate quota between devices and metadata, the test would be more useful if each of devices and metadata don't go over the quota separately but together they do.

Reply:
Added a separate test for that.